src/prototype_room_my.js

```diff
@@ -128,13 +128,14 @@ Room.prototype.handleObserver = function() {
 
     // TODO scan full range, first implementation
     const nameSplit = this.splitRoomName();
-    const fullLength = 2 * OBSERVER_RANGE + 1;
+    const observerRange = (config.room.observerRange <= 10 && config.room.observerRange > 0) ? config.room.observerRange : OBSERVER_RANGE;
```

Comment:
`config.room.observerRange <= OBSERVER_RANGE` instead? in case it changes, or is different on a private server. also nice even if only to remove magic numbers.

Reply:
also, if config.room.observerRange somehow gets set to -1, do we really want to bump it to 10 instead of to 0?

Reply:
`const observerRange = Math.min(config.room.observerRange, OBSERVER_RANGE)`

Reply:
just min() ignores the possibility that config.room.observerRange is negative. should we just say that's an error condition, or should we handle that gracefully here if something else has programmatically set that config lower than 0?

Reply:
why not : 
`config.room.observerRange === -1 ? OBSERVER_RANGE : Math.min(config.room.observerRange, OBSERVER_RANGE);`

Reply:
`config.room.observerRange < 0 ? OBSERVER_RANGE : Math.min(config.room.observerRange, OBSERVER_RANGE);`

Reply:
why set it up to a value negative if we want 0 ? just setup directly 0 ^^ we could be able to set it up to -1 for use constant one

Reply:
I am predicting a scenario where someone uses code to calculate their desired config.room.observerRange dynamically, and that code unexpectedly sets it to an arbitrary negative number.

I would rather use `observerRange: OBSERVER_RANGE` in a config if I want the constant one.

Reply:
ok but shouldn't be difficult to even cap the result of any getObserverRange method. agree for use constant in config directly ^^. then just a Math.min should be enouth

Reply:
I wouldn't do any magic. If `< 0` set to `OBSERVER_RANGE`.